src/containers/VisitPage.js

```diff
@@ -345,7 +320,7 @@ class VisitPage extends Component {
          * point. Perhaps a user's page should be a route
          * in VisitPage? Like our form sections will be? */}
         { this.state.redirect ? (
-          <Redirect to={ `/detail/${this.state.clientInfo.clientId}` } />
+          <Redirect to={ `/detail/${this.state.clientId}` } />
```

Comment:
This is `this.props.clientId` now, right?

Reply:
Yes! Good catch!